dependencies.yaml

```diff
@@ -565,6 +562,31 @@ dependencies:
               - pylibwholegraph-cu11==25.2.*,>=0.0.0a0
           - {matrix: null, packages: [*pylibwholegraph_unsuffixed]}
 
+  depends_on_librmm:
```

Comment:
We probably want the same for other packages too? libcudf, libraft? It’s okay to defer on this in the name of gradual improvement.

Reply:
Sure, we can go further in this PR. I just pushed https://github.com/rapidsai/cugraph/pull/4805/commits/64d0fad48f3d07e6100eea13d008d4de985a7280 moving more of the RAPIDS dependencies into these `depends_on_*` lists. 

Some notes on other changes in that commit:

* proposes removing explicit `libraft-headers` listing
   - `libraft` is explicitly listed, and directly pulls that in ([recipe link](https://github.com/rapidsai/raft/blob/d7e68f55c58493bc9cf1fbe4eb775a9593891c58/conda/recipes/libraft/meta.yaml#L157))
   - normally I'd advise against relying on transitive relationships like this implicitly, but in this case I think it's ok... the `libraft` conda package's purpose is to pull in the compiled bits of RAFT and the headers needed to link against them
   - if you disagree I'll happily add a `depends_on_libraft_headers`, don't feel THAT strongly about it either way
* removes `pyproject` and `requirements` entries for `libcudf`, `libraft,` and `librmm`
  - because `libcudf` is only used for benchmarks here, which aren't built from wheels
  - because `libraft` wheels don't actually exist yet
  - because `librmm` wheels aren't being used here yet
  - I'll add `libraft` and `librmm` entires in #4804
  - these `requirements` dependencies would all end up in the devcontainers environment: https://github.com/rapidsai/devcontainers/blob/branch-25.02/features/src/rapids-build-utils/opt/rapids-build-utils/bin/make-pip-dependencies.sh#L46

Requesting a re-review, that's too much to merge on a stale approval.

Reply:
Yes, it is okay to use libraft to get its headers transitively.

The other bits are fine, too. I’ll re-review, just make sure update-version.sh and conda recipes reflect things correctly.

Reply:
Alright great, thanks.

> update version.sh and conda recipes

Just did that in https://github.com/rapidsai/cugraph/pull/4805/commits/34697a69b91ff5b65b1556a3b8a557dba58674f9.

* removed `libraft-headers` from conda recipes and `update-version.sh`
* put a floor on `thriftpy2` in `dependencies.yaml` to match what conda recipes were using